test/system/test_cvp_change_control_api.py

```diff
@@ -0,0 +1,784 @@
+                self.cc_id)
+            assert approval_get_one is not None
+            assert approval_get_one['value']['approve']['value'] is True
+            assert approval_get_one['value']['approve']['notes'] == APPROVE_NOTE
+            assert approval_get_one['value']['key']['id'] == self.cc_id
+            time.sleep(1)
+
+            # verify with actual api output
+
+            response = self.get_cc_status(
+                self.cc_id)
+
+            assert response is not None
+            assert approval_get_one['value']['key']['id'] == response['value']['key']['id']
+            assert approval_get_one['value']['approve'][
+                'notes'] == response['value']['approve']['notes']
+            assert approval_get_one['value']['approve'][
+                'value'] == response['value']['approve']['value']
+
+            # Delete change control
+            self.delete_change_control(self.cc_id)
+
+            # Cancel Task
+            self.cancel_task(task_id)
+
+    def test_api_change_control_approval_get_one_without_approve(self):
+        """ Verify change_control_approval_get_one_without_approve
+         """
+        pprint(
+            "test_api_change_control_approval_get_one_without_approve")
+        if self.get_version():
+            # Create task
+            task_id = self.create_task()
+
+            # Create CC
+            self.create_change_control_for_tasks(
+                task_id)
+
+            pprint(
+                'APPROVAL GET ONE WITHOUT APPROVE...')
+            approval_get_one = self.api.change_control_approval_get_one(
+                self.cc_id)
+            assert approval_get_one is None
+
+            # Delete CC
+            self.delete_change_control(self.cc_id)
+
+            # Cancel Task
+            self.cancel_task(task_id)
+
+    def test_api_change_control_create_for_empty_tasks_list(self):
+        """ Verify change_control_create_for_tasks for empty task list
+        """
+        pprint(
+            "test_api_change_control_create_for_empty_tasks_list")
+        if self.get_version():
+            # Set client apiversion if it is not already set
+            pprint(
+                'RUN TEST FOR V3 CHANGE CONTROL APIs')
+            with self.assertRaises(CvpRequestError):
+                self.create_change_control_for_tasks(
+                    [])
+
+    def test_api_change_control_create_for_none_task_id_in_list(self):
+        """ Verify change_control_create_for_tasks for none task id in list
+        """
+        # Set client apiversion if it is not already set
+        pprint(
+            "test_api_change_control_create_for_none_task_id_in_list")
+        if self.get_version():
+            pprint(
+                'CREATE CHANGE CONTROL FOR LIST OF NONE TASK IDs...')
+            with self.assertRaises(CvpRequestError):
+                self.create_change_control_for_tasks([
+                                                     None])
```

Comment:
Same as above.

Reply:
As mentioned this test seems almost the same as above given the list type parameter sent to create_change_control_for_tasks is not valid.

Reply:
The difference only in invalid task_id passed to the function.